pdks/FinFET14nm_Mock_PDK/generators.json

```diff
@@ -1,12 +1,14 @@
 {
     "RES": "Res",
     "CAP": "Cap",
-    "Switch_NMOS_B": "Switch_NMOS_B",
-    "Switch_PMOS_B": "Switch_PMOS_B",
-    "Switch_NMOS_G": "Switch_NMOS_G",
-    "Switch_PMOS_G": "Switch_PMOS_G",
-    "NMOS": "Switch_NMOS",
-    "PMOS": "Switch_PMOS",
+    "NMOS_S": "NMOS",
+    "PMOS_S": "PMOS",
+    "NMOS_B": "NMOS",
+    "PMOS_B": "PMOS",
+    "NMOS_G": "NMOS",
+    "PMOS_G": "PMOS",
+    "NMOS": "NMOS",
+    "PMOS": "PMOS",
```

Comment:
Yes. I will remove this file soon with the cell generator update of the multiple expect ratio. This will require some changes as the return statement from the cell generator need to be handled properly.